### DISCUSSION TITLE
SEnding custom positioning system positions to the crazyflie

### DISCUSSION THREAD

**Discussion by ar-ramchandra** [bitcraze/discussions#1713] [Q&A] [answered]
Hi, 
I'm looking at creating my own positioning system for indoor use using cameras, if I am able to localize the 3d position of the crazyflie, and want to send it over to the drone from a computer, using the crazy PA or radio, is this do able ? if so could someone pls share any guidence on where i can find materials to read and implement the position transfer? what protocol and format i should use to send out the position...

Please and thank you in advance.

**Comment by varadVaidya** [comment, marked as answer]
You can use `cflib` to send the external position to crazyflie. The API for this is [here].(https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/api/cflib/crazyflie/extpos/)

You might have to set the corresponding paramters of the standard deviation of the position estimate through the parameters in [this group.](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/params/#locsrv)

**Reply by ar-ramchandra** [reply]
Hello,
Thank you very much for your quick response ! 

My idea is to use my external positioning system to send positions to the crazyflie, and have skybrush control the drones for indoor light shows..

If I use the crazy radio and an independent computer to send out position updates, would I need another crazy radio on another computer running skybrush ?  Or would they all work together on the same radio ?

**Reply by ToveRumar** [reply]
Hi @ar-ramchandra !

Im not sure I completely understand your question, but Ill try to give you an answer as best as I can. 

You need a crazyradio connected to the computer that is controlling the Crazyflies. You can use one radio to control several Crazyflies, we recommend around 3 Crazyflies per radio.  

Did this answer your question?
